glide.lock

```diff
@@ -133,12 +135,70 @@ imports:
 - name: github.com/docker/distribution
   version: 325b0804fef3a66309d962357aac3c2ce3f4d329
   subpackages:
+  - context
   - digest
   - reference
+  - registry/api/errcode
+  - registry/api/v2
+  - registry/client
+  - registry/client/auth
+  - registry/client/auth/challenge
+  - registry/client/transport
+  - registry/storage/cache
+  - registry/storage/cache/memory
+  - uuid
 - name: github.com/docker/docker
```

Comment:
Do we need all of thoses subpackages ? (anyway.. never really understood how glide handled them 😝)